wdn/templates_5.3/includes/global/head-2.html

```diff
@@ -1,25 +1,18 @@
-<link rel="preload" href="https://unlcms.unl.edu/wdn/templates_5.3/css/critical.css?dep=$DEP_VERSION$" as="style">
-<link id="unl-css-critical" rel="stylesheet" href="https://unlcms.unl.edu/wdn/templates_5.3/css/critical.css?dep=$DEP_VERSION$">
-<link id="unl-css-main" rel="preload" href="https://unlcms.unl.edu/wdn/templates_5.3/css/main.css?dep=$DEP_VERSION$" as="style" onload="this.onload=null;this.rel='stylesheet';">
-<link rel="preload" href="https://unlcms.unl.edu/wdn/templates_5.3/fonts/liberator/liberator-heavyitalic.woff2" as="font" type="font/woff2" crossorigin>
-<link rel="preload" href="https://cloud.typography.com/7717652/7106412/css/fonts.css" as="style" onload="this.onload=null;this.rel='stylesheet'">
+<link rel="preload" href="/wdn/templates_5.3/css/critical.css?dep=$DEP_VERSION$" as="style">
```

Comment:
shouldn't this file be from unlcms.unl.edu ?  same with the reference a couple lines down